.github/workflows/generate-pr-design-docs.yml

```
@@ -0,0 +1,105 @@
+          HTML_FILE="docs/design/pr-designs/pr-${PR_NUM}.html"
+
+          # Check if there are changes to commit
+          if git diff --quiet "$MD_FILE" "$HTML_FILE" 2>/dev/null; then
+            echo "No design doc changes to commit — doc already up to date."
+          else
+            git config user.name "github-actions[bot]"
+            git config user.email "github-actions[bot]@users.noreply.github.com"
+            git add "$MD_FILE" "$HTML_FILE"
+            git commit -m "[agento] docs: add design doc for PR #${PR_NUM}
+
+Generated by generate-pr-design-docs.yml workflow
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)